spec/index.bs

```diff
@@ -2577,6 +2595,10 @@ interface GPUAdapter {
     ::
         Returns the value of {{GPUAdapter/[[adapter]]}}.{{adapter/[[fallback]]}}.
 
+    : <dfn>isXrCompatible</dfn>
```

Comment:
Just curious, is this something applications need to be able to query?

`isFallbackAdapter` is actually exposing some new data (tells you if got a fallback adapter or not, in particular when you did not ask for `forceFallbackAdapter), whereas this just reflects the request option.

Reply:
You're right, it's not really important to be able to query it. Which actually means that we want to keep the internal slot instead of making the other one into a slot backed attribute. [1]